robotframework_reportportal/service.py

```diff
@@ -34,10 +34,15 @@
 def to_epoch(date):
     """Convert Robot Framework timestamp to UTC timestamp."""
     try:
-        iso_date = isoparse(date)
+        iso_date = parse(date)
     except ValueError:
-        return ''
-    return str(int(iso_date.timestamp() * 1000))
+        return None
```

Comment:
Please, update .pyi file. In our case, it does not matter, but in general, I would recommend returning values of one type within the same funtion.

Reply:
OK.
But since we do this in `Launch` class:
```python
self.start_time = attributes.get('starttime')
```
It's also possible that `start_time` will be `None`. So I also updated `to_epoch` method input argument, and `Suite` class field.